maps/aurora/aurora-6_surface.dmm

```diff
@@ -5247,7 +5247,7 @@
 "jy" = (
 /obj/machinery/door/airlock/security{
 	name = "Security Checkpoint";
-	req_access = list(1)
+	req_access = list(63)
```

Comment:
Undocumented change? What does this accomplish?

Reply:
All mapping changes should be properly documented now.